src/Covid19Dashboard/PlotChart/index.jsx

```diff
@@ -40,20 +60,48 @@ function formatChartData(plots) {
   });
   let sortedData = Object.values(dateToData);
   sortedData = sortedData.sort((a, b) => new Date(a.date) - new Date(b.date));
-  // console.log(sortedData);
-  return { data: sortedData }; // , max };
+  return { data: sortedData, ticks: getDates(sortedData[0].date, sortedData[sortedData.length - 1].date, 7) }; // , max };
 }
 
+// 10 colors generated with
+// https://medialab.github.io/iwanthue/
+// Intense + colorblind option + hard (force vector)
+const COLORS = ["#46151f", "#0078b8", "#00beb5", "#f5bc6e", "#006ad0", "#ff50a2", "#009b2b", "#91c600", "#6266ff", "#ff3cca",];
```

Comment:
what do you think about making the colors configurable in each plot? so that if COLORS[0] and COLORS[4] go better together than COLORS[0] and COLORS[1], we can choose that
`props.plots` could contain `{ name, data, color }` objects

Reply:
I changed the color palette to a default color palette from matplotlib. It looks much nicer and can address this.